src/components/badge/examples/badge-string.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * String badges
+ *
+ * Strings get truncated if they take up too much space
+ * (defined as ...). Hovering the cursor on the truncated
+ * badges shows the full text.
```

Comment:
> Hovering the cursor on the truncated badges shows the full text.


This is not how it works yet. So we have to add this with the next commit, when we add this functionality.